template/package.json

```diff
@@ -12,6 +12,7 @@
     },
     "dependencies": {},
     "devDependencies": {
+        "@vertigis/web": "5.7.0-beta.96",
```

Comment:
This shouldn't be necessary as the `create` script adds the dependency for you.

Reply:
Oops, removed.